bin/post_compile

```diff
@@ -11,7 +11,7 @@ MANAGE_FILE=${MANAGE_FILE:2}
 # Run migrations
 
 echo "-----> Running django migrations"
-python $MANAGE_FILE migrate --list 2>&1 | indent
+python $MANAGE_FILE showmigrations --list 2>&1 | indent
```

Comment:
Seems that way, yes. https://gist.github.com/singingwolfboy/7a12076520eccf19851ae00f461f5ab2